clang/lib/Basic/Targets/NVPTX.h

```diff
@@ -143,6 +143,7 @@ class LLVM_LIBRARY_VISIBILITY NVPTXTargetInfo : public TargetInfo {
     // FIXME: maybe we should have a way to control this ?
     Opts.support("cl_khr_int64_base_atomics");
     Opts.support("cl_khr_int64_extended_atomics");
+    Opts.support("cl_khr_3d_image_writes");
```

Comment:
Should we add a test for it?

Reply:
This PR is enables opencl capabilities that are already tested. There is not specific support for image in clang/llvm for NVPTX, we are adding this specifically for SYCL.